README.md

```diff
@@ -23,3 +23,17 @@ List of the [GitHub](https://github.com) usernames of all the people interested
 - [Shilpa](https://github.com/shilpavijay)
 
 This list is not yet complete—we'll keep adding more people.
+
+# Funding ideas
```

Comment:
WDYT about Rails Summer of Code?
https://railsgirlssummerofcode.org/

Reply:
for the name sounds like working on pandas will be out of scope, but if it's not, happy to get it in the list in a follow up PR

Reply:
Well, in the past they have accepted projects in all languages and not just Rails.
But, for the time being they are ungoing a revamp. Maybe we could revisit them later.
https://railsgirlssummerofcode.org/blog/2019-03-21-the-future-of-rgsoc

Reply:
That's good to know, then surely worth having it. Do you mind creating an issue to add it as a follow up once this is merged?